### PR TITLE
Map overview

### PR DESCRIPTION
**Pull request by slimonslimon**
This Improvement of Browse page add small pre-generated image of map with GPS track. 
It can improve orientation in logs on browse page. 
Overview image is generated when user visits plot_app page for first time and the image is saved in data/img direcotry.

### CODE REVIEW

**Comment by bkueng** [comment]
Awesome! Really neat.
A few notes:
- can you use the cache directory, similar for the kml:
  - https://github.com/PX4/flight_review/blob/master/plot_app/config.py#L90
  - and make sure to create it at setup-time: https://github.com/PX4/flight_review/blob/master/setup_db.py#L27
- can you center the image vertically in the column?
- in my test the map generation takes >4 seconds, which is quite long and I'm wondering if we can either reduce that and/or run it asynchronously or when idle? Also I would limit it to logs that actually appear on the browse page (public logs).

**Comment by slimonslimon** [comment]
Thank you for your sugesstions!

But I have one question about general concept of preview image: I planned to have preview image generated for all saved .ulog files - it is generated when someone visit the plot page. So browse page should be loaded as fast as possible independet to generation time. (I will write script which generate preview images for all existing logs which can be run when server idle)
Do you have better idea?

Other suggestions are clear, i will implement them.

**Comment by slimonslimon** [comment]
the map is now generatd faster.

Other problem is that Im not able to get continues integration tests to work, because im not familiar with pipenv and I have only python 3.5

File: generate_overview_img.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (bkueng) [inline generate_overview_img.py]: Can you make this script executable? -> `chmod +x generate_overview_img.py`.

Comment (slimonslimon) [reply generate_overview_img.py]: Of course

**Comment by slimonslimon** [comment]
Ok, It is solved by generating other html, when preview image doesnt exist.

I did not found any solution with CSS - for aligement I need display: block - but it is applied to alt text too.

**Comment by slimonslimon** [comment]
Im stil not able to get Pipfile compatible with requirementes.txt

**Comment by slimonslimon** [comment]
Thank you for your help.

**Pull request merged**